source/help/apps/desktop-guide.rst

```diff
@@ -0,0 +1,187 @@
+===================================
+Desktop Application's End User's Guide
+===================================
+
+Mattermost desktop applications are available for Windows, Mac and Linux operating systems. 
+
+You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and you can visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up.
+
+For latest updates, please see our `changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_.
+
+Below are a few tips to get you started and configuring your experience on the desktop app:
+
+ - `Team Management <https://docs.mattermost.com/help/apps/desktop-guide.html#team-management>`_
+ - `Settings <https://docs.mattermost.com/help/apps/desktop-guide.html#settings>`_
+ - `Menu Bar <https://docs.mattermost.com/help/apps/desktop-guide.html#menu-bar>`_
+
+Team management
+---------------------------------------------------------------------
+
+You can connect to multiple Mattermost servers from a single interface on the desktop app.
+
+Adding servers and teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To add a new server on the Mattermost instance, 
+
+1. Click **Add new team** next to the right of Team Management section.
+2. Enter **Name** and a valid **URL**, which begins with either ``http://`` or ``https://``.
+3. Click **Add**.
+
+You can now access all teams you have joined in the server once you have saved the settings, and each will appear as a separate tab in the app. 
+
+You may also add a separate instance for each team, if you'd like.
+
+Editing servers and teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To edit a server, 
+
+1. To the right of the team or server you want update, click **Edit**.
+2. Edit **Name** and/or **URL**.
+3. Click **Save**.
+
+Removing servers and teams
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To remove a server from your desktop app environment, click on **Remove** next to the server or a team you want removed. Note that this action cannot be undone.
```

Comment:
Saying "action cannot be undone" seems to imply that you can't add it back in once you remove it - is there another way of saying this? To me, there doesn't seem to be any consequence of removing a team/server.